src/index.ts

```diff
@@ -29,7 +29,8 @@ function defaultSettings(settings: Partial<Settings>): Settings {
       sortPropertiesByName: true,
       commentEverything: false,
       ignoreFiles: [],
-      indentationChacters: '  '
+      indentationChacters: '  ',
+      honorCastTo: [],

```

Comment:
```suggestion
      honorCastTo: ['string', 'number'],
```
I would be fully happy to make this defaulted to the types you are supporting. Could even go so far as to make it a private setting that cannot be changed

Reply:
Apologies on my late response, but that sounds good! I was aiming to maintain backward compatibility initially, but really I think the standard use case will be to want this. That being said, I have updated the README indicating that only sting and number cast types are supported currently. Still working on the set and map PR, but I should be pushing that up tomorrow I believe.